src/sentry/api/endpoints/group_events.py

```diff
@@ -81,6 +81,7 @@ def _get_events_snuba(self, request, group, environments, query, tags, start, en
         params = {
             "group_ids": [group.id],
             "project_id": [group.project_id],
+            "organization_id": group.project.organization,
```

Comment:
Should this be `group.project.organization_id`?

Reply:
:+1: good catch, still find it a bit odd that Django is very okay with `.filter(organization_id=Organization)`

Reply:
Yeah django can turn the object into its pk value, but you pay for the extra DB query/association hydration.